### ISSUE TITLE
num_classes Bug[Bug]: 

### ISSUE THREAD

**Issue by shamaseen** [tinyvision/DAMO-YOLO#65]
### Before Reporting

- [X] I have pulled the latest code of main branch to run again and the bug still existed. 我已经拉取了主分支上最新的代码，重新运行之后，问题仍不能解决。

- [X] I have read the [README](https://github.com/tinyvision/DAMO-YOLO/blob/master/README.md) carefully and no error occured during the installation process. (Otherwise, we recommand that you can ask a question using the Question template) 我已经仔细阅读了README上的操作指引，并且在安装过程中没有错误发生。（否则，我们建议您使用Question模板向我们进行提问）

### Search before reporting

- [X] I have searched the DAMO-YOLO [issues](https://github.com/tinyvision/DAMO-YOLO/issues) and found no similar bugs. 我已经在[issue列表](https://github.com/tinyvision/DAMO-YOLO/issues)中搜索但是没有发现类似的bug报告。

### OS

Ubuntu

### Device

Nvidia V100

### CUDA version

10.2,

### Python version

3.10

### PyTorch version

1.7.0

### torchvision version

0.13.1

### Describe the bug

I have tried to train the model on a dataset with 1 class, i change 'num_classes' :1 on [damoyolo_tinynasL20_T.py]
but give me this warning. 
` No Positive Samples on cuda:0! May cause performance decrease. loss_bbox:0.000, loss_dfl:0.000, loss_qfl:0.307 `

But when I change it to  'num_classes':52 the training start, and the dataset has 1 class but I need set  'num_classes'  to 52.

### To Reproduce

1- add dataset with 1 class
2- [DAMO-YOLO/configs/damoyolo_tinynasL20_T.py](https://github.com/tinyvision/DAMO-YOLO/blob/6e38813220900955d0f6138429c91a33a79c922f/configs/damoyolo_tinynasL20_T.py#L64-L66)

Lines 64 to 66 in [6e38813](https://github.com/tinyvision/DAMO-YOLO/commit/6e38813220900955d0f6138429c91a33a79c922f)

 ZeroHead = { 
     'name': 'ZeroHead', 
     'num_classes': 1,
3-python -m torch.distributed.launch --nproc_per_node=8 tools/train.py -f configs/damoyolo_tinynasL20_T.py

**Comment by cwhgn** [comment]
Thanks for your attention. The warning indicates that the samples in current batch have no labels. "num_classes=1 or 52" has nothing to do with this phenomenon. You can ignore the warning during your training and check whether the model is trained as expected. We have checked that  the phenomenon would cause little effect to the performance under normal circumstances.
